test/integration/default/controls/_mapdata_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# Prepare platform "finger"
+platform_name = case platform[:name]
+                when 'amazon'
+                  'amazonlinux'
+                else
+                  platform[:name]
+                end
```

Comment:
As taught to us by @n-rodriguez, we can get more Ruby-ish!

```suggestion
platform_name =
  case platform[:name]
  when 'amazon'
    'amazonlinux'
  else
    platform[:name]
  end
```

Reply:
Ok, sometime it may avoid the line too long issue ;-)